### PR TITLE
CEDS-2667 Add HMRCReportTechnicalssue component

### PR DESCRIPTION
**Pull request by PauloPortugal**
This component is a non-ajax implementation and is the recommended
approach by the HMRC Play UI team, since the previous "GetHelpLink" was
not accessible.

This component does not work locally and it should simply open a new
page. We have been told that this will work in all the other
environments.

### CODE REVIEW

File: app/views/components/gds/gds_main_template.scala.html [hunk 1]
```diff
@@ -63,6 +68,7 @@
 
   @if(useTimeoutDialog) {
       <script src='@routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
+      <script>window.HMRCFrontend.initAll();</script>
```
Comment (russellthorn) [inline app/views/components/gds/gds_main_template.scala.html:71]: 👍

File: app/views/components/gds/gds_main_template.scala.html [hunk 2]
```diff
@@ -89,6 +95,12 @@
         )))
     </div>
     @contentBlock
+    @hmrcReportTechnicalIssue(
+        ReportTechnicalIssue(
+            serviceCode = appConfig.keyStoreSource,
```
Comment (glidester) [inline app/views/components/gds/gds_main_template.scala.html:100]: Nit: would be nicer if 'keyStoreSource' was renamed to 'appName' like in internal-frontend. Seems odd having a persistence var here (i know values are the same)

**Review by glidester — APPROVED** [review]

File: app/views/components/gds/gds_main_template.scala.html (hunk 2 above)
Comment (PauloPortugal) [reply app/views/components/gds/gds_main_template.scala.html]: @glidester your wish has come true.

Comment (PauloPortugal) [reply app/views/components/gds/gds_main_template.scala.html]: Resolved

**Review by glidester — APPROVED** [review]

**Pull request merged**